COLLABORATOR_GUIDE.md

```diff
@@ -388,13 +432,13 @@ This means that Shiki's Themes and Languages are not bundled by default.
 
 Hence the `shiki.config.mjs` file, where we define our custom set of supported Languages and we bundle them directly by using [Shiki's Grammar Property](https://github.com/shikijs/shiki/blob/main/docs/languages.md#supporting-your-own-languages-with-shiki) which allows us to embed the languages directly.
 
-### Tailwind
+### TailwindCSS
 
-Tailwind is a great CSS Framework. It allows us to create a Design System that is easy to maintain and extend. It also allows us to create a consistent Design Language across the Website.
+[Tailwind][] is a great CSS Framework. It allows us to create a Design System that is easy to maintain and extend. It also allows us to create a consistent Design Language across the Website.
 
 For example, it automatically integrates with PostCSS and Next.js, which allows us to use Tailwind's utilities within our Components and Pages.
 
-It also integrates with Sass allowing to use Tailwind classes directly within our SCSS Modules.
+It also integrates with Sass allowing to use Tailwind classes directly within our CSS Modules.
```

Comment:
Sass reference that might not make sense. please review

Reply:
Feel free to remove!